prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -870,10 +871,10 @@ abstract class CoolTest {
                 ?: error("Unable to find sender $fullELRSenderName for organization ${org1.name}")
         }
 
-        const val etorTISenderName = "ignore-etor-ti"
-        val etorTISender by lazy {
-            settings.findSender("$org1Name.$etorTISenderName") as? UniversalPipelineSender
-                ?: error("Unable to find sender $etorTISenderName for organization ${org1.name}")
+        const val fhirSenderName = "ignore-fhir-e2e"
```

Comment:
There were a few variables declared here that weren't being used, so I cleaned them up.